### PR TITLE
fix(components): [el-popper] optimize popper performance with disable eventlistener when invisible

### LINKED ISSUE #6125: [Bug Report] el-tooltip组件append-to参数无效，是否内部参数为传递给el-tooltip-content组件
### Element Plus version
2.0.1

### OS/Browsers version
chrome 98.0.4758.80

### Vue version
3.2.16

### Reproduction Link
https://jsfiddle.net/1df3yuhs/

### Steps to reproduce
el-tooltip组件设置append-to属性值为#app，鼠标移动至按钮上面

### What is Expected?
内容显示在#app节点下面

### What is actually happening?
内容显示在body节点下面，content.vue文件未接收到appendTo参数

### LINKED ISSUE #6152: [Bug Report] el-table-column使用插槽放置el-popover会造成页面异常卡顿
### Element Plus version
2.0.2

### OS/Browsers version
Chrome： 98.0.4758.102 

### Vue version
3.2.31

### Reproduction Link
https://codesandbox.io/s/elementplus-tps9q4?file=/src/App.vue

### Steps to reproduce

  1.打开codesandbox链接，遇到构建错误请刷新查看

  2.模拟数据量增加，请连续点击Add item

  3.会明显发现拖动的时候异常卡顿，如对应真实环境分页50条/页，已经没法操作

### What is Expected?
卡顿消失

### What is actually happening?
数据量大的时候卡顿非常明显，我看了下Performance的性能摘要，发现Rendering的时间少则300ms 多则接近快一秒，而且滚动条连续拖动触发scroll event会频繁调用Forceupdate与之对应则是FunctionforceUpdate @ createPopper.js:142:40 看起来好像是el-popover的锅？

### PR DESCRIPTION
**Pull request by weidehai**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.
fix #6152  bug

### CODE REVIEW

File: packages/components/popper/src/content.vue [hunk 1]
```diff
@@ -96,6 +104,8 @@ export default defineComponent({
             })
             popperInstanceRef.value = newInstance
 
+            watch(() => props.visible, togglePopoerAlive, { immediate: true })
+
```
Comment (JeremyWuuuuu) [inline packages/components/popper/src/content.vue:108]: Why are we using nested watch? I don't think this is a good practice, can we move it outside this watch call? We can watch it independently after this watch.

File: packages/components/tooltip/src/content.vue [hunk 2]
```diff
@@ -10,6 +10,7 @@
         v-if="shouldRender"
         v-show="shouldShow"
         ref="contentRef"
+        :visible="shouldShow"
```
Comment (JeremyWuuuuu) [inline packages/components/tooltip/src/content.vue:13]: Can we put this line before `:z-index`?

**Review by jw-foss — COMMENTED** [review]
I like your idea but still I left some comments for you to resolve, feel free to ping me if you have any questions at all ;)

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
@weidehai Thank you for making Element Plus better! :bow:

**Pull request merged**